docs/installation/network-options.md

```diff
@@ -209,11 +220,31 @@ systemctl restart k3s
 ```
 :::
 
-:::caution Warning
-The latency between nodes will increase as external connectivity requires more hops. This will reduce the network performance and could also impact the health of the cluster if latency is too high.
-:::
+### Tailscale
 
-:::caution Warning
-Embedded etcd will not use external IPs for communication. If using embedded etcd; all server nodes must be reachable to each other via their private IPs.
-:::
+Available in v1.27.3, v1.26.6, v1.25.11 and newer.
+
+K3s can integrate with [tailscale](https://tailscale.com/) so that nodes use the tailscale VPN network to communicate with each other.
+
+There are four steps to be done with tailscale before deploying k3s:
+1 - Create an account in tailscale
+2 - In Settings > Keys, generate an auth key (<auth-key>), which may be reusable for all nodes in your cluster
```

Comment:
Does this need to be unique per cluster? Can multiple clusters use the same key? Are there any other considerations for having multiple clusters on a single account?